src/styles/SearchResults.css

```diff
@@ -0,0 +1,45 @@
+.card-img {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 20%;
+}
+
+.search-results__no-results {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 50vh;
+  text-align: center;
+  color: #c70019;
+  font-size: 2rem;
+  padding: 2rem;
+}
+.search-results {
```

Comment:
Adding a clear line between class definitions would make this a bit easier to read